src/Elastic.Apm/Report/Serialization/TrimmedStringJsonConverter.cs

```diff
@@ -1,12 +1,19 @@
 using System;
+using System.Runtime.CompilerServices;
```

Comment:
I think this is unused, could you please remove it?

Reply:
fixed